build/dependencies.props

```diff
@@ -6,57 +6,55 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Health Checks Package Versions">
```

Comment:
I think only 1 new line is needed here:
`
<HealthCheckAzureApplicationInsights>6.0.2</HealthCheckAzureApplicationInsights>
`

Reply:
do you mean I should revert this file and just add `<HealthCheckAzureApplicationInsights>6.0.2</HealthCheckAzureApplicationInsights` ?

Reply:
Of course. It is very strange to see that due to adding one new project, absolutely all lines are changed.

Reply:
I resolved this conflict for you.